app/controllers/signup_controller.rb

```diff
@@ -25,10 +21,12 @@ def start
     if request.post?
```

Comment:
I suspect the `GET` of `start` needs to be able to handle if we already know user role or user email?